web/angular-wallet/src/app/main/transactions/transaction.service.ts

```diff
@@ -50,20 +53,18 @@ export class TransactionService {
     }
 
     public async sendMoney({ transaction, pin, keys, recipientAddress }: SendMoneyRequest) {
-        // todo
-        // if (transactionToSend.attachment && transactionToSend.attachment.encryptedMessage) {
-        //     const recipientPublicKey = await getAccountIdFromPublicKey(transaction.recipientAddress);
-        //     const encryptedMessage = await encryptMessage(transactionToSend.attachment.encryptedMessage,
-        //         keys.agreementPrivateKey, this.accountService.hashPinEncryption(pin), recipientPublicKey);
-        //     transactionToSend = {
-        //         attachment: new EncryptedMessage({
-        //             data: encryptedMessage.m,
-        //             nonce: encryptedMessage.n,
-        //             isText: true
-        //         }),
-        //         ...transactionToSend
-        //     };
-        // }
+        if (transaction.attachment && (<EncryptedMessage>transaction.attachment).data) {
```

Comment:
didn't know, that sendMoney is encrypted, too... oh, if I send money with message, the message could be encrypted... I think I got it

Reply:
yup sendMoney supports both un/encrypted msgs